### ISSUE TITLE
mongodb link

### ISSUE THREAD

**Issue by atherdon** [ChickenKyiv/api-extended-database#5]
`mongodb://heroku_b97rxdzw:fnnua12mbtlqd5bh0i35roo0a@ds141524.mlab.com:41524/heroku_b97rxdzw`

---

You'll need to have a loopback-connector-mongo package in order to setup it well.

Sample: https://github.com/GroceriStar/groceristar/blob/master/server/datasources.json

**Comment by khyati0212** [comment]
Let me if I am wrong but in tutorial it was not needed.

**Comment by atherdon** [comment]
Yeah, at this tutorial you need to install your own local mongodb server.
and i think this is the most weaker place of this tutorial

**Comment by atherdon** [comment]
instead of setup local database - I've created an online instance just for you

**Comment by khyati0212** [comment]
Oh great. Thanks.

**Comment by khyati0212** [comment]
I did install loopback-connector-mongo package
Can u send me automigrate sample details?

**Comment by atherdon** [comment]
this is the most recent version of import script that i did: https://github.com/ChickenKyiv/recipe-search-api/tree/master/bin

how to run this script: https://github.com/ChickenKyiv/recipe-search-api/blob/master/package.json#L15

not remember - but it can be broken at this moment. If it will crash - buzz me - and i'll finish it - cause i'm lazy to do it. 

nice progress, BTW. we'll need an upload stuff for images soon

**Comment by atherdon** [comment]
I think this ticket can be closed, right?

**Comment by khyati0212** [comment]
yes

**Comment by atherdon** [comment]
you need to use that link in order to connect mlab with our code
